packages/myst-cli/src/transforms/outputs.ts

```diff
@@ -37,7 +37,7 @@ export async function transformOutputsToCache(
 ) {
   const outputs = selectAll('output', mdast) as GenericNode[];
   // This happens sooner for notebooks
-  if (!outputs.length || kind !== SourceFileKind.Article) return;
```

Comment:
@rowanc1 this transform now does not ignore articles.

Reply:
Makes sense -- we are really blurring the lines now. :)